semver.md

```diff
@@ -4,15 +4,15 @@ Semantic Versioning 2.0.0
 Summary
 -------
 
-Given a version number MAJOR.MINOR.PATCH, increment the:
+Given a version number `MAJOR.MINOR.PATCH`, increment the:
```

Comment:
The quotes are not needed around MAJOR.MINOR.PATCH and seem to be confusing.  The all-caps seems to be sufficient.